internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -89,31 +114,62 @@ func (df *Dockerfile) parse() error {
 	file, err := df.fs.Open(df.path)
 
 	if err != nil {
-		return fmt.Errorf("read dockerfile: %w", err)
+		return fmt.Errorf("read Dockerfile: %w", err)
 	}
 	defer file.Close()
 
-	scanner := bufio.NewScanner(file)
-	parsedDockerfile := parseFromScanner(scanner)
+	f, err := afero.ReadFile(df.fs, file.Name())
+	if err != nil {
+		return fmt.Errorf("read Dockerfile %s error: %w", f, err)
+	}
+	fileString := string(f)
+
+	parsedDockerfile, err := parseFromReader(fileString)
+	if err != nil {
+		return fmt.Errorf("parse Dockerfile: %w", err)
+	}
 
 	df.ExposedPorts = parsedDockerfile.ExposedPorts
+	df.HealthCheck = parsedDockerfile.HealthCheck
 	df.parsed = true
 	return nil
 }
 
-func parseFromScanner(scanner *bufio.Scanner) Dockerfile {
-	var line = ""
+func parseFromReader(line string) (Dockerfile, error) {
```

Comment:
nit: 
```suggestion
func parse(content string) (Dockerfile, error) {
```

Reply:
One thing I'd love is for this function to return `(*Dockerfile, error)` and if there is an error return `nil, err`.

I find it a bit confusing that there are error scenarios where we need to read the content of `Dockerfile` and some where we don't.